### PR TITLE
[#8626] Term 'Recovery phrase' should be used instead of 'seed phrase'

### LINKED ISSUE #8626: Term 'Recovery phrase' should be used instead of 'seed phrase' for release 0.14.0
# Problem

In previous releases up to 0.13.0 we were using term 'recovery phrase' for 12 words used to recover an account. In the release 0.14.0 because of re-design some parts of the app use term 'seed phrase' for the same entity. It should be fixed so old users have consistent experience.

## Implementation

Places to change seed phrase to recovery phrase:
1. Account explorer:
Screenshot_20190724-120215

2. v1 warning
Screenshot_20190724-120250

## Acceptance Criteria

"Seed phrase" term is not shown on any screen in the app. Term "recovery phrase" is used instead of "seed phrase"

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #8626

### CODE REVIEW

**Review by vitvly — APPROVED** [review]

**Review by dmitryn — APPROVED** [review]

File: translations/en.json [hunk 1]
```diff
@@ -1223,15 +1223,15 @@
...
     "keycard-recovery-intro-text": "If you generated a key using a keycard before and now want to use this key on this device",
     "keycard-recovery-intro-button-text": "Begin recovery",
     "keycard-recovery-success-header": "Your key has been\n successfully recovered",
     "keycard-recovery-no-key-header": "There’s nothing to \nrecover here",
     "keycard-recovery-no-key-text": "Your Keycard has no key stored on it. In order to use it, generate a new key and choose your Keycard to store the key",
-    "keycard-recovery-phrase-confirmation-title": "Written the Seed phrase down?",
-    "keycard-recovery-phrase-confirmation-text": "You cannot do this later. Without the Seed phrase you will not be able to access your key or any assets associated with it if you loose your device.",
+    "keycard-recovery-phrase-confirmation-title": "Written the recovery phrase down?",
+    "keycard-recovery-phrase-confirmation-text": "You cannot do this later. Without the recovery phrase you will not be able to access your key or any assets associated with it if you loose your device.",
     "keycard-recovery-phrase-confirm-header": "Confirm “Seed” phrase",
```
Comment (dmitryn) [inline translations/en.json:1235]: Looks like `"Confirm “Seed” phrase"` should also be changed

**Review by dmitryn — APPROVED** [review]

File: translations/en.json (hunk 1 above)
Comment (flexsurfer) [reply translations/en.json:1235]: why is it “Seed” ?

Comment (dmitryn) [reply translations/en.json:1235]: Don't know, сopied from Figma as is

Comment (flexsurfer) [reply translations/en.json:1235]: @rachelhamlin @hesterbruikman ?

Comment (churik) [reply translations/en.json:1235]: @dmitryn From my POV, it should be changed as well for consistency inside app as it mentioned in https://github.com/status-im/status-react/issues/8626

**Comment by hesterbruikman** [comment]
Please keep changes to a minimum. In new designs we deliberately use 'seed phrase'. At first time mentioning it in onboarding notated as "seed" phrase. Some inconsistency for Beta 0.14 is fine, from v1 onwards, consistency should be on using seed phrase.  

Reason is that seed phrase is a eco system standard and already an abstraction layer. Consistency and the narrative is aimed at building understanding that if you have the seed you can grow your tree again or grow your existing tree somewhere else.

**Comment by flexsurfer** [comment]
@hesterbruikman i'm confused, should we change recovery phrase to seed phrase then ? cc @rachelhamlin

**Comment by annadanchenko** [comment]
keycard is not enabled in release 0.14.0 so we can keep new term there. 

However, I strongly insist to provide some transition period where we would mention recovery phrase term next to seed phrase. It's needed in the places where user recovers an account or gets instructions like v1 banner to save it. Something like "recovery phrase (aka seed phrase)" instead of just a new "seed phrase". Otherwise, novice users from 0.13.0 and lower versions will be really puzzled on where to get that seed phrase.

**Comment by rachelhamlin** [comment]
Right. `Seed phrase` is correct for anything that's not being merged into v0.14 @flexsurfer; we're just going to keep `recovery phrase` for consistency in beta.

Not sure I agree a transition period is necessary. That's mental overhead for a small change that most users are educated enough to understand, imo.

**Comment by churik** [comment]
checked for backup recovery phrase on IOS and Android.
Couldn't find a way to get v1 warning for PR build - made a manual release build https://ci.status.im/job/status-react/job/manual/419/, but after installing and recover account still don't get it.
According to https://github.com/status-im/status-react/issues/8626#issue-472194273 it should be changed.
Also I couldn't find in files that this text is changed.
Can you check it @flexsurfer?

**Comment by rachelhamlin** [comment]
Yes please, let's double check use of `recovery phrase` :)

**Comment by flexsurfer** [comment]
@churik v1 popup we have only in the release, it will be changed here https://github.com/status-im/status-react/pull/8632#issuecomment-514673877

**Pull request merged**